js/DataService.js

```diff
@@ -1,9 +1,23 @@
 angular.module('PathOfDamage')
 .service('DataService', function () {
-  var NAME_DELIMITER = '\r';
+  var VALUE_DELIMITER = '\r';
   var ROW_DELIMITER = '\f';
   var SECTION_DELIMITER = '\0';
 
+  var elementToIndex = {
+    fire: 0,
+    cold: 1,
+    lightning: 2,
+    chaos: 3
+  };
+
+  var indexToElement = {
+    0: 'fire',
+    1: 'cold',
+    2: 'lightning',
+    3: 'chaos'
+  };
```

Comment:
I think you could just make an array of the strings, then use indexOf('fire') to get the index, and square brackets to pull the string out using the index.